lona/server.py

```diff
@@ -447,9 +447,9 @@ async def _handle_http_request(self, http_request):
             return response
 
         # websocket requests ##################################################
-        if(http_request.method == 'GET' and
-           http_request.headers.get('upgrade', '').lower() == 'websocket'):
-
+        if (http_request.method == 'GET' and
+                http_request.headers.get('upgrade', '').lower() ==
+                'websocket'):
```

Comment:
This seems to be only a style change. Can you put this in another commit?

Reply:
Removed this change. It was done by my IDE, not intentionally.